tests/sql/src/main/java/sql/snappy/SnappyTest.java

```diff
@@ -137,6 +137,7 @@ public SnappyTest(SnappyNode snappyNode) {
   }
 
   public static synchronized void HydraTask_initializeSnappyTest() {
+    System.setProperty("gemfire.DISALLOW_RESERVE_SPACE", "true");
```

Comment:
Is this invoked on all nodes? Check with QA, otherwise better to have them set it in their common local.conf.

Reply:
Asked @spillai14 . She confirmed.

Will run some hydra and check.